pkg/nuget/lock/parse.go

```diff
@@ -49,7 +51,33 @@ func (p *Parser) Parse(r dio.ReadSeekerAt) ([]types.Library, []types.Dependency,
 				Version: packageContent.Resolved,
 			}
 			libs = append(libs, lib)
+
+			depId := utils.PackageID(packageName, packageContent.Resolved)
+			dependsOn := make([]string, 0)
```

Comment:
```suggestion
			var dependsOn []string
```
I think we can use this variable declaration